src/chalk-overview.md

```diff
@@ -0,0 +1,141 @@
+# An Overview of Chalk
+
+> Chalk is under heavy development, so if any of these links are broken or if
+> any of the information is inconsistent with the code or outdated, please open
+> an issue so we can fix it. If you are able to fix the issue yourself, we would
+> love your contribution!
+
+[Chalk][chalk] recasts Rust's trait system explicitly in terms of logic
+programming by "lowering" Rust code into a kind of logic program we can then
+execute queries against. Its goal is to be an executable, highly readable
+specification of the Rust trait system.[^negativechalk]
```

Comment:
I don't understand the footnote here?

Reply:
This description is a paraphrased (and copied) version of the blog post in the footnote so I thought I should cite my sources :stuck_out_tongue_closed_eyes: